src/org/robockets/stronghold/robot/turntable/Turntable.java

```diff
@@ -26,6 +26,9 @@
 	public final PIDController pidController;
 	public final EncoderPIDSource encoder;
 	
+	public final double factor = 0.5;
```

Comment:
This variable is never used. Probably also needs a comment explaining what it is.